RGB/ContentView.swift

```diff
@@ -8,12 +8,63 @@
 import SwiftUI
 
 struct ContentView: View {
+    @State private var redValue: Double = 128
+    @State private var greenValue: Double = 128
+    @State private var blueValue: Double = 128
+    @State private var rgbText: String = ""
+    
     var body: some View {
         VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundStyle(.tint)
-            Text("Hello, world!")
+            Text("RGB")
+                .font(.largeTitle)
+                .fontWeight(.bold)
+                .padding()
+            
+            RoundedRectangle(cornerRadius: 15)
+                .fill(Color(red: redValue / 255, green: greenValue / 255, blue: blueValue / 255))
+                .frame(height: 200)
+                .shadow(radius: 10)
+                .padding(.horizontal)
+            
+            Text("Adjust the sliders to mix your color")
+                .font(.subheadline)
+                .padding(.top)
+            
+            ColorSlider(value: $redValue, textColor: .red)
+            ColorSlider(value: $greenValue, textColor: .green)
+            ColorSlider(value: $blueValue, textColor: .blue)
+            
+            if !rgbText.isEmpty {
```

Comment:
I would just remove this check because when the string becomes non-empty, whole UI jumps to make a room for it.